### DISCUSSION TITLE
Drawing an Array

### DISCUSSION THREAD

**Discussion by serhatyaris34** [emgucv/emgucv#642] [Q&A] [answered]
Hi all, Is it possible to draw an arrow with Emgu like we do with System.Drawing.Drawing2D.AdjustableArrowCap?(as shown below)
```
            //BEFORE EMGU
            AdjustableArrowCap myArrow = new AdjustableArrowCap(2.5f, 2.5f);
            myPen = new Pen(Color.Red);
            myPen.CustomEndCap = myArrow;
```
![Github_emgu_arrow](https://user-images.githubusercontent.com/32990026/140317749-2f989319-988d-4d9b-b620-4d47ffbc248d.png)

```
            //Is it possible? If yes, how?
            LineSegment2DF lineSegment_obj = new LineSegment2DF(new PointF(200.0f, 200.0f),
                new PointF(300.0f, 200.0f));
            imf.Draw(lineSegment_obj, new Bgr(Color.Red), 1,
                     Emgu.CV.CvEnum.LineType.FourConnected, 0);
```

**Comment by emgucv** [comment, marked as answer]
You can use CvInvoke.ArrowedLine:
https://www.emgu.com/wiki/files/4.5.4/document/html/M_Emgu_CV_CvInvoke_ArrowedLine.htm
The corresponding Open CV function is:
https://docs.opencv.org/4.5.4/d6/d6e/group__imgproc__draw.html#ga0a165a3ca093fd488ac709fdf10c05b2

**Reply by serhatyaris34** [reply]
Thank you.
